qiskit/pulse/instructions/__init__.py

```diff
@@ -57,7 +57,7 @@
 """
 from .acquire import Acquire
 from .delay import Delay
-from .directives import Directive, RelativeBarrier
+from .directives import Directive, RelativeBarrier, TimeBlockade
```

Comment:
I know it's mostly for internal use, but did you want to add this to the autosummary for the docs above since you mention it in the release notes.

Reply:
Thanks. Done in https://github.com/Qiskit/qiskit-terra/pull/8949/commits/1756e35a9d7ed57101a934ccf0eb6148ba57a31c